source/contracts/reporting/Market.sol

```diff
@@ -293,28 +304,25 @@ contract Market is DelegationTarget, ITyped, Initializable, Ownable, IMarket {
         }
         _destinationUniverse.addMarketTo();
         _currentUniverse.removeMarketFrom();
-        IReputationToken _oldReputationToken = getReputationToken();
         universe = _destinationUniverse;
 
         universe.incrementOpenInterestFromMarket(_marketOI);
 
-        // reset state back to Initial Reporter
-        IInitialReporter _initialParticipant = getInitialReporter();
+        // Pay the No Show REP bond
```

Comment:
Due to disavowal, there is NO rep associated with this market or its participants any more.  The user that is performing this migrateThroughOneFork is will to either stake 
1.) NoShowBond (with no benefit to themselves) if the market has not reached endTime. 
2.) The InitialReport if the market HAS reached endTime. To do this, it does take a trip through the noShowBond, but it really has no effect, code was just structured this way to keep conditional logic clean

The "re-used" initialReport gets updated with the sender's address and starts using the new universe's rep.

If market has not reached endtime, the arguments to this function are completely unused (and noShowBond remains on the market). If it HAS reached endtime, it performs an initial report, using the noShowBond that was JUST put in place.